query-engine/connector-test-kit-rs/query-engine-tests/tests/new/ref_actions/on_delete/set_default.rs

```diff
@@ -69,10 +69,7 @@ mod one2one_req {
     }
 
     /// Deleting the parent reconnects the child to the default and fails (the default doesn't exist).
-    #[connector_test(
-        schema(required_with_default),
-        exclude(MongoDb, MySQL, Vitess("planetscale.js", "planetscale.js.wasm"))
-    )]
+    #[connector_test(schema(required_with_default))]
```

Comment:
So this means that the test is now passing on previously excluded "provider"? 🤔

Reply:
They are still failing, but I've moved the exclusion to the top.